projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.html

```diff
@@ -125,3 +125,27 @@
     *ngIf="hasError">
   </adf-empty-content>
 </aca-page-layout>
+
+<ng-template #skeleton>
+  <div class="adf-skeleton-container">
```

Comment:
Can this be done without extra container div? When answer is loading, this will replace html which is inside ng-container with loading ngIf so I believe that whole skeleton will be already wrapped in some element (like some div) after replacement, so seems redundant to add extra div. 

Maybe when you remove that segment with Agent then it will be easier to skip this container.